bin/run-prober

```diff
@@ -1,19 +1,14 @@
-#! /bin/bash
+#! /usr/bin/env bash
 
 # DOC: Run the browser test suite in Docker against the staging environment.
 
-pushd $(git rev-parse --show-toplevel)
-
-set -e
-set +x
+source bin/lib.sh
 
 docker run \
   -v "$(pwd)/browser-test/src:/usr/src/civiform-browser-tests/src" \
   -v "$(pwd)/browser-test/bin:/usr/src/civiform-browser-tests/bin" \
-  -e BASE_URL=https://staging.seattle.civiform.com \
-  -e TEST_USER_LOGIN=$TEST_USER_LOGIN \
-  -e TEST_USER_PASSWORD=$TEST_USER_PASSWORD \
+  -e BASE_URL="https://staging.seattle.civiform.com" \
```

Comment:
might be worthwhile to make this an env variable but also like fine not to do that yet

Reply:
Yeah, when we're ready to run this on both staging environments we'll probably split out two prober tasks